controller/comments-controller.js

```diff
@@ -0,0 +1,28 @@
+const Comment = require("../models/comment");
+const Posts = require("../models/posts");
+const Scripts = require("./scripts");
```

Comment:
This should be `scripts`
I've suggested `User` only for models.

Reply:
done